src/main/resources/io/viash/platforms/nextflow/VDSL3Helper.nf

```diff
@@ -584,6 +584,14 @@ def processFactory(Map processArgs) {
     outputPaths = outputPaths + ', path{[".exitcode"]}'
   }
 
+  // create dirs for output files (based on BashWrapper.createParentFiles)
+  def createParentStr = thisConfig.functionality.allArguments
+    .findAll { it.type == "file" && it.direction == "output" && it.create_parent }
+    .collect { par -> 
+      "\${ args.containsKey(\"${par.plainName}\") ? \"mkdir_parent \\\"\" + (args[\"${par.plainName}\"] instanceof String ? args[\"${par.plainName}\"] : args[\"${par.plainName}\"].join('\" \"')) + \"\\\"\" : \"\" }"
```

Comment:
Can the values here also be `Gstrings`? https://docs.groovy-lang.org/latest/html/api/groovy/lang/GString.html

Reply:
Not sure what you mean by this. 

Do you mean `it.create_parent` or `par.plainName` could be a GString' If so, then no.

Do you mean whether I should be using a gstring inside the collect? I am, though?

Or do you mean you noticed I'm using a GString, but you don't think it'll work? I think it works, though.

Either way, I'm confused :P

Reply:
Yes, I was referring to the fact that the config could contain Gstrings somewhere 🤷 But it can't, great!